whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/organisation/OrgNameAndCountryAttrValidator.java

```diff
@@ -84,14 +92,19 @@ private boolean isReferencedByRsMaintainedResource(final RpslObject rpslObject)
         }
         return false;
     }
+    private boolean isAttributeModified(final AttributeType attrType, final RpslObject originalObject, final RpslObject updatedObject) {
+        final CIString originalAttrValue = originalObject.getValueOrNullForAttribute(attrType);
+        final CIString updatedAttrValue = updatedObject.getValueOrNullForAttribute(attrType);
 
-    private boolean orgNameDidntChange(final RpslObject originalObject, final RpslObject updatedObject) {
-        final CIString originalOrgName = originalObject.getValueOrNullForAttribute(AttributeType.ORG_NAME);
-        final CIString updatedOrgName = updatedObject.getValueOrNullForAttribute(AttributeType.ORG_NAME);
+        if(originalAttrValue == null && updatedAttrValue == null) {
+            return false;
+        }
+
+        if(originalAttrValue == null || updatedAttrValue == null) {
+            return true;
+        }
 
-        return (originalOrgName != null) &&
-                (updatedOrgName != null) &&
-                (Objects.equals(originalOrgName.toString(), updatedOrgName.toString()));
+        return !(Objects.equals(originalAttrValue.toString(), updatedAttrValue.toString()));
```

Comment:
Or, restore the original logic, I think it's more readable
`
return (originalAttrValue != null) &&
(updatedAttrValue != null) &&
(Objects.equals(originalAttrValue.toString(), updatedAttrValue.toString()));
`

Reply:
Old logic will not work as org-name can never be null, it was not handling that case

Reply:
OK